minard/pmtnoisedb.py

```diff
@@ -12,8 +16,23 @@ def dictify(rows):
     return rv
 
 def get_noise_results(limit=100, offset=0):
+    epoch = datetime(1970, 1, 1)
     conn = engine_nl.connect()
     rows = conn.execute('SELECT * FROM pmtnoise '
                         'ORDER BY run_number DESC '
                         'LIMIT %s OFFSET %s;', (limit, offset))
+    rows = dictify(rows)
+    for row in rows:
+        row['display_time'] = str(row['timestamp'])[:-6]
+        delta = (row['timestamp'].replace(tzinfo=None) - row['timestamp'].utcoffset() - epoch)
+        plot_time = delta.days*SECOND_PER_DAY
+        plot_time += delta.seconds
+        plot_time += delta.microseconds*SECOND_PER_MICROSECOND
+        row['plot_time'] = plot_time
+    return rows
```

Comment:
Sorry, should have mentioned earlier. If you do:

```python
from .tools import total_seconds
```

it will give you an equivalent function. I think I added that a while ago because I hd the same issue.

If you can make this change, then squash all the commits, I'll merge. Thanks!

Reply:
Very nice --- ready to go.